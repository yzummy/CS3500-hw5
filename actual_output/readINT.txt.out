HERE 2222
the 2 boolean value here is FALSE"Enter an integer (e.g., 123): "

---- Completed parsing ----

Value of the expression is: 
