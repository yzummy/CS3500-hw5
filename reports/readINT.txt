							      >	"Enter an integer (e.g., 123): "

---- Completed parsing ----					---- Completed parsing ----

							      >	Value of the expression is: 123
