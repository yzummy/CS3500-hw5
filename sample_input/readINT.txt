# The expected output file expects you to enter 123
{
  print("Enter an integer (e.g., 123): ");
  read()
}
