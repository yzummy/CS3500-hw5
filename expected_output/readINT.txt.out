"Enter an integer (e.g., 123): "

---- Completed parsing ----

Value of the expression is: 123
